src/Surfnet/ServiceProviderDashboard/Domain/Entity/Entity/MetaData.php

```diff
@@ -79,14 +82,22 @@ public static function fromApiResponse(array $data)
 
         Assert::string($entityId);
         Assert::string($metaDataUrl);
-        Assert::string($acsLocation);
+        Assert::allString($acsLocations);
         Assert::string($nameIdFormat);
         Assert::string($certData);
         Assert::string($descriptionEn);
         Assert::string($descriptionNl);
         Assert::string($nameEn);
         Assert::string($nameNl);
 
+        if (count(array_unique($acsLocations)) !== count($acsLocations)) {
+            throw new Exception('Double acs locations. Expected unique locations');
+        }
+
+        if (count($acsLocations) > self::MAX_ACS_LOCATIONS) {
+            throw new Exception('Maximum acs locations exceeded. Maximum 10 asc location are supported');
+        }
```

Comment:
I love you added these input validations! 

1. Must we fail so hard on these conditions? Or would a log message be good enough? Or maybe just ignore the excessive items, or.. One to run by @quartje ?
2. There is a 'lil typo in the second exception a~s~cs

Reply:
After consultation with Bart. Manage does not validate whether two of the same ACS locations exist. It is useful to perform that validation client side on our side.